test/acceptance/user.acceptance.ts

```diff
@@ -43,7 +47,11 @@ describe('User (acceptance)', () => {
     request = supertest(`http://127.0.0.1:${port}`);
   }
 
-  async function givenUser(data: Partial<User>) {
+  async function givenCreatedUser(data: Partial<User>) {
     return await repository.create(data);
   }
+
+  async function givenFoundUser(name: string) {
```

Comment:
Building on top of my previous comment, this function is not used in test setup, therefore its name should not start with `given` prefix. 

Here is a better name: `findUserByName()`.